Common/Securities/CashBuyingPowerModel.cs

```diff
@@ -118,6 +125,8 @@ public override HasSufficientBuyingPowerForOrderResult HasSufficientBuyingPowerF
                     GetMaximumOrderQuantityForTargetBuyingPower(
                         new GetMaximumOrderQuantityForTargetBuyingPowerParameters(parameters.Portfolio, parameters.Security, targetPercent)).Quantity * GetOrderPrice(parameters.Security, parameters.Order);
 
+                maximumQuantity = Math.Round(maximumQuantity, amountPrecision);
```

Comment:
`GetMaximumOrderQuantityForTargetBuyingPower` is already using the lotSize to determine the correct quantity, this shouldn't be required or it's a bug in the `GetMaximumOrderQuantityForTargetBuyingPower` implementation

Reply:
Hm ok I will need to add tests that reproduce the issues but based on the code I can already see an issue, the code below does not respect the lot size, it does not round after multiplication has been performed
![image](https://user-images.githubusercontent.com/33373431/103611038-16857d80-4f2a-11eb-9b0c-9bb9d5218237.png)